Task 'ide' not found in root project 'spotless'

Hi all 👋  I'm new here and would love to start contributing, but I ran into the following while setting up the IDE for development.

The [CONTRIBUTING.md](https://github.com/diffplug/spotless/blob/main/CONTRIBUTING.md#contributing-to-spotless) guide mentions twice to run `gradlew ide` to setup the Eclipse IDE for development. But this results in the following error message:

```
./gradlew ide

FAILURE: Build failed with an exception.

* What went wrong:
Task 'ide' not found in root project 'spotless'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 589ms
```

Perhaps the guide is out of date, but I'd like to know how to get started. Generally, I code in IntelliJ-IDEA, but I don't mind having to switch to Eclipse for this. I'm also not used to using gradle (only maven), so perhaps I just do something wrong 😄 

## Context
- macOS 12.0.1
- Gradle 6.6
- db06b285822ee036289a3db1def5216c76891846

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -1,6 +1,6 @@
 # Contributing to Spotless
 
-Pull requests are welcome, preferably against `main`.  Feel free to develop spotless any way you like, but the easiest way to look at the code is to clone the repo and run `gradlew ide`, which will download, setup, and start an Eclipse IDE for you.
+Pull requests are welcome, preferably against `main`.  Feel free to develop spotless any way you like.
 
 ## How Spotless works
 
@@ -36,7 +36,6 @@ For the folders below in monospace text, they are published on maven central at 
 | `lib-extra` | Contains the optional parts of Spotless which require external dependencies.  `LineEnding.GIT_ATTRIBUTES` won't work unless `lib-extra` is available. |
 | `plugin-gradle` | Integrates spotless and all of its formatters into Gradle. |
 | `plugin-maven` | Integrates spotless and all of its formatters into Maven. |
-| ide | Generates and launches an IDE for developing spotless. |
 | _ext | Folder for generating glue jars (specifically packaging Eclipse jars from p2 for consumption using maven).
 
 ## How to add a new FormatterStep

